source/server/server.cc

```diff
@@ -46,8 +46,8 @@ InstanceImpl::InstanceImpl(Options& options, Network::Address::InstanceConstShar
                                POOL_GAUGE_PREFIX(stats_store_, "server."))},
       thread_local_(tls), api_(new Api::Impl(options.fileFlushIntervalMsec())),
       dispatcher_(api_->allocateDispatcher()), singleton_manager_(new Singleton::ManagerImpl()),
-      handler_(new ConnectionHandlerImpl(log(), *dispatcher_)), listener_component_factory_(*this),
-      worker_factory_(thread_local_, *api_, hooks),
+      handler_(new ConnectionHandlerImpl(logger(), *dispatcher_)),
```

Comment:
Can you wrap this in new macro, ENVOY_GET_LOGGER() or something.

Reply:
done